ratpack-cassandra/src/main/java/smartthings/ratpack/cassandra/CassandraModule.java

```diff
@@ -98,6 +100,10 @@ public Boolean getAutoMigrate() {
 			return autoMigrate;
 		}
 
+		public Boolean isSpeculativeExecutionEnabled() {
```

Comment:
Don't we need getter/setter so we can make this false in some cases?

Reply:
fixed